std_msgs/Header header
uint8 msg_type
uint8 msg_length
uint8[] msg_body

